docs/examples_benchmark.md

```diff
@@ -0,0 +1,68 @@
+
+### PNEUMONIA
+<p align="center">
+    <img src="./pics/pneumonia_time.jpg" alt="pneumonia training time" width="550"/>
+</p>
+For pneumonia, FL takes only 4% longer wall-clock time than centralized-1/3, and about 5% longer computing time than centralized-1.
+
+### NER
+<p align="center">
+    <img src="./pics/ner_time.jpg" alt="ner training time" width="600"/>
+</p>
+For ner, FL takes only 1.3% longer wall time than centralized-1/3, and only 0.1% longer computing time than centralized-1.
+
+### CCFRAUD
+<p align="center">
+    <img src="./pics/cc_time.jpg" alt="ccfraud training time" width="600"/>
+</p>
+For ccfraud, FL takes 10% longer wall time than centralized model, while about 3% longer computing time than centralized-1.
+
+## Model Performance
+Another important assessing factor for FL is the model performance. Here we also aim at two questions: 1. How does the FL model performance compare to the centralized model trained with only partial data, which is the scenario when FL is not supported and data are confidential and restricted to each region. 2. How does the FL model performance compare to the centralized model trained with data from all silos, which is an ideal situation when all data are eyes-on and could be combined. This is to understand if the distributed-aggregate design of FL has impact on the model performance. For this reason, we tested 3 models: "FL" for model trained with FL in 3 silos, "Centralized-1/3" for 1 model with 1/3 data, and "Centralized-1" for 1 model with all data combined. 
+
+### PNEUMONIA
+<p align="center">
+    <img src="./pics/pneumonia_acc.jpg" alt="pneumonia model performance" width="550"/>
+</p>
+For pneumonia, FL achieves higher accuracy than centralized-1/3, while slightly lower than centralized-1.
+
+### NER
+<p align="center">
+    <img src="./pics/ner_acc.jpg" alt="ner training time" width="600"/>
+</p>
+For ner, FL achieves a highest score for all four metrics. Although it is not expected that FL will outperform centeralized-1, it might be becasue the distribute-aggregate fashion improves the generalizability of the final model.
+
+
+## Scalability with Training
+
+For each real-world example, we also support distributed training within each silo, empowered by Pytorch Distributed Data Parallel (DDP) module. To test the scalability of our implementation, we artifically replicated each datasets by 10 times, and record the training time per epoch for each silo when such data is trained on different number of GPUs.
```

Comment:
I would put more emphasis here, example:

Scalability is critical for industry applications of FL on large datasets. One benefit of using FL on Azure ML is that it supports distributed training (multi gpu and multi node). To test the scalability of our implementation....

Then add a highlight before the details: in all 3 scenarios, we can achieve scalability by adding more nodes and gpus to reduce wall time accordingly.